src/Scrive/Form/Validator.elm

```diff
@@ -0,0 +1,80 @@
+module Scrive.Form.Validator exposing (..)
+
+import Either exposing (Either(..))
+import Scrive.Data.Address as A
+import Scrive.Data.Tag as Tag exposing (SomeTag, Tag)
+import Scrive.Form.Error as FE
+import Scrive.Form.Field as Field
+import Set
+import Validate as V exposing (Validator)
+import Validate as Validate
```

Comment:
NIT/Q: I didn't know you can alias one module with multiple names. Is this intentional? Also what is the reason to `Validate as Validate`?